Fix grammar and phrasing (#223)
PyCQA/redbaron: README.md

[![Backers on Open Collective](https://opencollective.com/redbaron/backers/badge.svg)](#backers) 
[![Sponsors on Open Collective](https://opencollective.com/redbaron/sponsors/badge.svg)](#sponsors) 

> "RedBaron is a Python library and tool powerful enough to be used in IPython. 
> It aims to make the process of writing code that modifies source code as easy and simple as possible." That includes writing custom

Author: ```suggestion
"RedBaron is a Python library designed to be used with IPython. 
It aims to make the process of writing code that modifies source code as easy and simple as possible." That includes writing custom
```

This was trying to be as close to original as possible. At the same time it now seems that first sentence tells nothing in particular except association with IPython. Is RedBaron designed to work best with IPython?

> refactoring, generic refactoring, tools, IDE or directly modifying you source

Author: what is the difference of custom and generic refactoring?
